src/net_defs.h

```diff
@@ -142,6 +142,15 @@ typedef enum
     NET_MASTER_PACKET_TYPE_SIGN_END_RESPONSE,
 } net_master_packet_type_t;
 
+typedef enum {
```

Comment:
At the very least this doesn't belong here. Define a header in midiproc/ and put it there.